dhusget_wrapper.sh

```diff
@@ -145,11 +145,18 @@ if [[ -z ${path_product_list:-} ]]; then
 
     #limit the number of scenes when footprint set to a
     #small high latitude region by searching around the local solar noon +-dt hours
-    if ! [[ ${footprint} == "Greenland" ]]; then
-      
-      y=${footprint_poly:22:-2}
-      dt=1  	
-      output=`python <<END
+    if ! [[ ${footprint} == "Greenland" ]] && ! [[ ${footprint} == "AlaskaYukon" ]]; then
```

Comment:
In the case of Greenland or AlaskaYukon it looks like dt is not set. Is that right?

Reply:
No indeed, there is no limitation for Greenland and AlaskaYukon. Actually there shouldn't be one for Iceland neither (I will change it). 
We implemented this `dt` variable (few years ago) as a way to limit the number of scenes for high latitude regions but I think we should directly limit the number of scenes so that it is not region specific. Actually maybe indirectly it is not, but I will need to check.

Reply:
ah ok I now see that datestr0 and datestr1 were set just above for the general case:
```
    datestr0="${date}T00:00:00.0000Z"
    datestr1="${date}T23:59:59.9999Z"
```

all good!